lib/cisco.py

```diff
@@ -999,7 +999,11 @@ def __str__(self):
         # add a header comment if one exists
         for comment in header.comment:
           for line in comment.split('\n'):
-            target.append(' remark %s' % line)
+            if filter_type == 'standard':
+              target.append(' access-list %s remark %s' % (filter_name,
+                                                           line))
```

Comment:
Please put this on one line since it doesn't go over the line limit of 80 characters